### PR TITLE
Enhance README's readability & translated

### PR DESCRIPTION
**Pull request by Mr-Sheep**
- Added section to run Caddy as a daemon and made a clear distinction between server-side and client-side
- Translated README to Simplified Chinese
- Enhanced readability(a little bit) of README

### CODE REVIEW

**Comment by Mr-Sheep** [comment]
Apologize for missing one line when running Caddy as a daemon; since I use daemon configuration from official Caddy dpkg, they created a new user called "caddy" to run Caddy as a daemon. 
One will have to create a new user called "caddy" using `useradd caddy`

**Comment by xh116** [comment]
Firstly, you really did a good job and made much efforts.
But I do not think it is necessary to translate it to Chinese. 
I found all other tools like shadowsocks/v2ray/trojan designed for GFW have no chinese [README]. 
Not sure why, maybe a small barrier for some "naive" guys ? 

By the way, naiveproxy's about is "make fortune quietly"... 

Maybe you can PR to wiki or a tutorial page but not the main page.

File: README.md [hunk 1]
```diff
@@ -15,45 +17,89 @@ The code consists of a patch series morphed and rebased onto every new Chrome re
 
 [Browser → Naïve (client)] ⟶ Censor ⟶ [Frontend → Naïve (server)] ⟶ Internet
 
-NaïveProxy uses Chrome's network stack. The traffic behavior intercepted by the censor is identical to regular HTTP/2 traffic between Chrome and standard Frontend (e.g. Caddy, HAProxy).
+NaïveProxy uses Chrome's network stack. Thus, the traffic behavior intercepted by the censor is identical to regular HTTP/2 traffic between Chrome and standard Frontend (e.g., Caddy, HAProxy).
```
Comment (klzgrad) [inline README.md:20]: Comma after e.g. is optional https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e.

File: README.md [hunk 2]
```diff
@@ -15,45 +17,89 @@ The code consists of a patch series morphed and rebased onto every new Chrome re
...
   }
   file_server { root /var/www/html }
 }
 ```
-`:443` must appear first for this Caddyfile to work. For more advanced usage consider using JSON for Caddy 2's config.
+`:443` must appear first for this Caddyfile to work. For more advanced usage consider [using JSON for Caddy 2's config](https://caddyserver.com/docs/json/).
+
+#### Run Caddy as a daemon
+
+To do so, edit `/lib/systemd/system/caddy.service ` with the following content:
```
Comment (klzgrad) [inline README.md:66]: No, you can't create files under /lib directly.

File: README.md [hunk 3]
```diff
@@ -63,43 +109,43 @@ Locally run `./naive` with the following `config.json` to get a SOCKS5 proxy at
...
 See [USAGE.txt](https://github.com/klzgrad/naiveproxy/blob/master/USAGE.txt) for more parameters in `config.json`. See also [Performance Tuning](https://github.com/klzgrad/naiveproxy/wiki/Performance-Tuning).
 
-## Build
+## Build form source
 
 If you don't like to download binaries, you can build NaïveProxy.
 
 Prerequisites:
-* Ubuntu (apt-get install): git, python2, ninja-build (>= 1.7), pkg-config, libnss3-dev, curl, unzip, ccache (optional)
+* Linux (apt-get install): git, python2, ninja-build (>= 1.7), pkg-config, libnss3-dev, curl, unzip, ccache (optional)
```
Comment (klzgrad) [inline README.md:117]: Can't claim Linux here. No other distros are tested.

File: README.md [hunk 4]
```diff
@@ -63,43 +109,43 @@ Locally run `./naive` with the following `config.json` to get a SOCKS5 proxy at
...
 
-Previously, Tor tried to mimic Firefox's TLS signature and still got [identified and blocked by firewalls](https://groups.google.com/d/msg/traffic-obf/BpFSCVgi5rs/nCqNwoeRKQAJ), because that signature was of an outdated version of Firefox and the firewall determined the rate of collateral damage would be acceptable. If we use the signature of the most commonly used browser the collateral damage of blocking it would be unacceptable.
+Previously, Tor tried to mimic Firefox's TLS signature and still got [identified and blocked by firewalls](https://groups.google.com/d/msg/traffic-obf/BpFSCVgi5rs/nCqNwoeRKQAJ) because that signature was an outdated version of Firefox. The firewall determined the rate of collateral damage would be acceptable. If we use the signature of the most commonly used browser, the collateral damage of blocking it would be unacceptable.
 
 ### Why not use Go, Node, etc. for performance?
 
-Any languages can be used for high performance architectures, but not all architectures have high performance.
+Any language can be used for high performance architectures, but not all architectures have high performance.
```
Comment (klzgrad) [inline README.md:140]: Any is usually followed by plural nouns.

File: README.md [hunk 5]
```diff
@@ -63,43 +109,43 @@ Locally run `./naive` with the following `config.json` to get a SOCKS5 proxy at
...
+Go, Node, etc. make it easy to implement a 1:1 connection proxy model, i.e. creating one upstream connection for every user connection. Then under this model, the goal of the performance is to reduce overhead in setting up each upstream connection. Toward that goal, people start to reinvent their own 0-RTT cryptographic protocols (badly) as TLS goes out of the window because it either spends take several round trips in handshakes or makes it [a pain to set up 0-RTT properly](https://tools.ietf.org/html/rfc8446#section-8). Then people also start to look at low level optimization such as TCP Fast Open.
```
Comment (klzgrad) [inline README.md:142]: The leading phrase is too short to require a comma.

File: README.md [hunk 6]
```diff
@@ -63,43 +109,43 @@ Locally run `./naive` with the following `config.json` to get a SOCKS5 proxy at
...
 
-And the real reason Google never enabled TCP Fast Open by default is that it was dragged down by middleboxes and [never really worked](https://blog.donatas.net/blog/2017/03/09/tfo/). In Linux kernel there is a sysctl called `tcp_fastopen_blackhole_timeout_sec`, and whenever a SYN packet is dropped, TCP Fast Open is blackholed for this much time, starting at one hour and increasing exponentially, rendering it practically useless. Today TCP Fast Open accounts for [0.1% of the Internet traffic](https://ieeexplore.ieee.org/document/8303960/), so using it actually makes you highly detectable!
+And the real reason Google never enabled TCP Fast Open by default is that it was dragged down by middleboxes and [never really worked](https://blog.donatas.net/blog/2017/03/09/tfo/). In Linux kernel, there is a sysctl called `tcp_fastopen_blackhole_timeout_sec`, and whenever a SYN packet is dropped, TCP Fast Open is blackholed for this much time, starting at one hour and increasing exponentially, rendering it practically useless. Today TCP Fast Open accounts for [0.1% of the Internet traffic](https://ieeexplore.ieee.org/document/8303960/), so using it actually makes you highly detectable!
```
Comment (klzgrad) [inline README.md:148]: Leading phrase too short

Comment (klzgrad) [inline README.md:148]: Leading phrase too short

**Comment by klzgrad** [comment]
1. I adjusted README per some of your suggestions. I won't other suggestions now.
2. The systemd service file, you need to create a separate file, not in README, and fix its content.
3. The Chinese README can go to a new page in https://github.com/klzgrad/naiveproxy/wiki. I have opened edit access.

File: README.md [hunk 7]
```diff
@@ -15,45 +17,89 @@ The code consists of a patch series morphed and rebased onto every new Chrome re
...
+Description=Caddy
+Documentation=https://caddyserver.com/docs/
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+User=caddy
+Group=caddy
+ExecStart=/usr/bin/caddy run --environ --config /etc/caddy/Caddyfile
+ExecReload=/usr/bin/caddy reload --config /etc/caddy/Caddyfile
```
Comment (klzgrad) [inline README.md:79]: It's problematic. Where to put the caddy binary, there is no fixed location to use here.

**Comment by Mr-Sheep** [comment]
> The systemd service file, you need to create a separate file, not in README, and fix its content.
> 
> The Chinese README can go to a new page in https://github.com/klzgrad/naiveproxy/wiki. I have opened edit access.

Two new wiki pages are created for running caddy as a daemon and Chinese README

**Comment by klzgrad** [comment]
Issue is completed.

**Comment by klzgrad** [comment]
I sent a collaborator invite but then considered it not appropriate. If wiki access is needed in the future I'll open it again temporarily.

**Comment by Mr-Sheep** [comment]
Understood, thanks for creating such a nice project

**Pull request closed without merging**